### ISSUE TITLE
Download Album Covers Along Songs!

### ISSUE THREAD

**Issue by Treion** [fast4x/RiMusic#4267]
### Description

While using the RiMusic app to download songs, I noticed that when offline, the album covers do not load, which detracts from the overall experience. Having a visual representation of the music enhances enjoyment, and I believe adding an option to download album covers along with songs would greatly improve user satisfaction.

### Suggested Solution

I propose implementing a new section in the settings dedicated to downloading preferences. This section could include:

- **Toggle Option for Album Covers**: A feature that allows users to choose whether to download album covers along with their songs.

- **Quality Selection for Album Covers**: Users should be able to select the quality of album cover images they wish to download.

- **Download Preferences**: Options for users to set their download preferences, including:

i) Downloading on Wi-Fi only or allowing cellular downloads.
ii) Notifications for completed downloads.
iii) Refresh rate settings for downloaded content.

This would provide users with a customizable experience tailored to their preferences.

### Alternatives

If implementing this feature is not feasible, an alternative could be to display a default album cover when offline, ensuring that users still have a visual representation of their music even without downloaded covers. Additionally, offering a curated selection of popular album covers for offline use could enhance the experience without requiring individual downloads.

**Comment by Treion** [comment]
Also I forgot another thing, If y'all ever plan to make that  new section in the settings all about downlaoads, Make an option to also download lyrics alongside the song

**Comment by JoshGMR** [comment]
I totally agree, I also asked for that feature a while ago as it would be very useful to be able to choose the covers to download in addition to the lyrics and not just see a black background with green cover
